packages/cos-setup/cos-setup-initramfs.service

```diff
@@ -10,7 +10,7 @@ RootDirectory=/sysroot
 BindPaths=/proc /sys /dev /run /tmp
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=/usr/bin/elemental run-stage initramfs
+ExecStart=-/usr/bin/elemental run-stage initramfs
 
 [Install]
 RequiredBy=initrd.target
```

Comment:
This could be an issue too. Honestly I do not remember why I added this as a hard requirement. `WantedBy` looks llike a better approach to me now